tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "limited time. Now finally let us perform some molecular dynamics:\n",
+    "\n",
+    "`% gmx grompp -c em.gro -p ../setup/topol.top -f /home/shared/data/md.mdp -maxwarn 1 -o md.tpr`\n",
+    "\n",
+    "`% cp /home/shared/data/submit_md.sh .`\n",
+    "\n",
+    "`% sbatch submit_md.sh`\n",
+    "\n",
+    "You can check the status of the job again as described previously.\n",
+    "\n",
+    "At the moment it is set up to run for 1000 ps. This will take several minutes to complete depending on the waiting time in the queue - time for lunch! You don't have to wait for it to finish completely though, although now might be a good time for a break to allow at least some data to appear. The analysis can be done on the output files that will be generated or you can always use the \"one I made earlier\" in the directory /home/shared/prerun/run (This is 1000 ps simulation of the same system).\n",
+    "\n",
+    "After the end of the production run, it would be useful to obtain some properties that will give us an insight into our protein system.\\\n",
+    "There are various so-called ensembles that are used for protein simulations - probably the most common is a system where the number of particles, the pressure and the temperature are held constant (NPT). This is usually achieved by means of a heat-bath. Nevertheless, it is usually a good idea to check these as a function of time through the trajectory just to make sure nothing unexpected happened. First let us check the temperature of our simulation.\n",
+    "\n",
+    "`% gmx energy -f md.edr -s md.tpr -o 1hsg_temperature.xvg`\n",
+    "\n",
+    "The program will then present you with a large table of all the values recorded in the energy (.edr) file. We want to examine temperature so type 13, press enter and then 0 and press enter again. The program will then analyse the temperature and present some statistics of the analysis at the end.\n",
+    "\n",
+    "Another set of properties that is quite useful to examine is the various energetic contributions to the energy. The total energy should be constant. but the various contributions can change and this can sometimes indicate something interesting or strange happening in your simulation. Let us look at some energetic properties of the simulation.\n",
+    "\n",
+    "`% gmx energy -s md.tpr -f md.edr -o 1hsg_energies.xvg`\n",
+    "\n",
+    "We shall select short-range lennard-jones (7), short range coulombic (9) and the potential energy (11). End your selection with a zero.\n",
+    "\n",
+    "Finally, we need to renumber the em.gro file so that the residues of the two chains will not have the same residue number. This will be necessary for when we calculate the RSF values.\n",
+    "\n",
+    "`% gmx editconf -f em.gro -o em_renumbered.gro -resnr 1`\n",
+    "\n",
+    "We will plot and explore the temperature and the energetic components that we obtained in the next section of the tutorial.\n",
```

Comment:
Might have missed it somewhere, but we should get them to trjconv their system to make sure it stayed in the box? It's unlikely to have moved that much in 1 ns, but still possible. If we don't do this here, we'll have to do it later using MDAnalysis, which is also ok to do, but a little bit more complex in terms of number of python lines.

Reply:
Updated. They are asked to modify the trajectory to place the protein at the center and remove rotational and translational motion.